src/providers/VoiceFocusProvider/docs/VoiceFocusProvider.stories.mdx

```diff
@@ -1,8 +1,12 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import { VoiceFocusProvider } from '../';
+
 <Meta title="SDK Providers/VoiceFocusProvider" />
 
 # VoiceFocusProvider
 
-The `VoiceFocusProvider` provides a function transforming a normal device to an Amazon Voice Focus device, and whether Amazon Voice Focus is supported.
+The `VoiceFocusProvider` provides a function transforming a normal device to an Amazon Voice Focus device, and whether Amazon Voice Focus is supported. 
+Amazon Voice Focus related logs can be found in the dev tool when the feature is enabled.
```

Comment:
nit: "in the browser developer tools when the Amazon Voice Focus is enabled".

Reply:
sure, will update